### ISSUE TITLE
Create example project

### ISSUE THREAD

**Issue by donovanr** [AllenCellModeling/datastep#1]
An example project using this + the cookiecutter template.
First idea i had was to use MNIST data set and a linear DAG:
- `raw` = download MNIST data and push to quilt
- `qc` = throw out some of the images for some manufactured arbitrary reason, e.g pixel intensity variance is too high, etc
- `norm` = normalize the images somehow, maybe transforming to RGB for use with a pytorch pretrained network
- `split` = make holdout / train split
- `train_classifier` = fine-tune a pytorch resnet18 or something
- `evaluate_classifier` = make precision/ recall / confusion matrix plots on held out data

**Comment by donovanr** [comment]
working on this here: https://github.com/AllenCellModeling/mti_mnist
